src/components/dls/Forms/Combobox/index.tsx

```diff
@@ -0,0 +1,265 @@
+/* eslint-disable jsx-a11y/role-has-required-aria-props */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, {
+  MouseEvent,
+  ChangeEvent,
+  useState,
+  ReactNode,
+  useEffect,
+  RefObject,
+  useCallback,
+  useRef,
+} from 'react';
+import classNames from 'classnames';
+import useOutsideClickDetector from 'src/hooks/useOutsideClickDetector';
+import useKeyPressedDetector from 'src/hooks/useKeyPressedDetector';
+import useScroll from '../../../../hooks/useScrollToElement';
+import CaretIcon from '../../../../../public/icons/caret-down.svg';
```

Comment:
no need to address it in this PR, I think that you can do `public/icons/...` directly since we configured the root in typescript